r_bin.h:341: error: redefinition of typedef 'RBin' - CentOS 6

https://github.com/radareorg/radare2/runs/835250871?check_suite_focus=true#step:5:294
```
/radare2/radare2/radare2/libr/include -fvisibility=hidden -I../../shlr/sdb/src//src -DHAVE_FORK=1 -I../../shlr -o str.o str.c
In file included from str.c:6:
/__w/radare2/radare2/radare2/libr/include/r_bin.h:341: error: redefinition of typedef 'RBin'
/__w/radare2/radare2/radare2/libr/include/r_bin_dwarf.h:802: note: previous declaration of 'RBin' was here
make[2]: *** [str.o] Error 1
make[2]: Leaving directory `/__w/radare2/radare2/radare2/libr/util'
make[1]: *** [all] Error 2
make[1]: Leaving directory `/__w/radare2/radare2/radare2/libr/util'
make: *** [all] Error 2
```

--- a/.github/workflows/centos6.yml
+++ b/.github/workflows/centos6.yml
@@ -32,7 +32,7 @@ jobs:
       run: make install
       working-directory: radare2
     - name: Run tests
-      run: cd test/unit && make && make fuzz-tests
+      run: cd test/unit && make
       working-directory: radare2
       env:
         # `make install` installs, for some unknown reasons, pkgconfig files in

--- a/libr/include/r_bin.h
+++ b/libr/include/r_bin.h
@@ -6,6 +6,9 @@
 #include <r_io.h>
 #include <r_cons.h>
 #include <r_list.h>
+
+typedef struct r_bin_t RBin;
+
 #include <r_bin_dwarf.h>
 #include <r_pdb.h>
 
@@ -302,7 +305,7 @@ typedef struct r_bin_file_options_t {
 	// const char *xtrname;
 } RBinFileOptions;
 
-typedef struct r_bin_t {
+struct r_bin_t {
 	const char *file;
 	RBinFile *cur; // TODO: deprecate
 	int narch;
@@ -338,7 +341,7 @@ typedef struct r_bin_t {
 	bool use_xtr; // use extract plugins when loading a file?
 	bool use_ldr; // use loader plugins when loading a file?
 	RStrConstPool constpool;
-} RBin;
+};
 
 typedef struct r_bin_xtr_metadata_t {
 	char *arch;

--- a/libr/include/r_bin_dwarf.h
+++ b/libr/include/r_bin_dwarf.h
@@ -799,8 +799,6 @@ typedef struct {
 
 #define r_bin_dwarf_line_new(o,a,f,l) o->address=a, o->file = strdup (f?f:""), o->line = l, o->column =0,o
 
-typedef struct r_bin_t RBin; // forward declaration so I can keep the functions in this interface
-
 R_API RBinDwarfDebugAbbrev *r_bin_dwarf_parse_abbrev(RBin *a, int mode);
 R_API RList *r_bin_dwarf_parse_aranges(RBin *a, int mode);
 R_API RBinDwarfDebugInfo *r_bin_dwarf_parse_info(RBinDwarfDebugAbbrev *da, RBin *a, int mode);

